kinto/api-test/test_kinto_heartbeat.py

```diff
@@ -19,11 +19,9 @@ def test_check_heartbeat(self):
         """Check heartbeat to make sure It's Alive."""
         resource = '__heartbeat__'
         response = self.client.get_request(resource, status_code=503)
-        self.assertIn('oauth', response)
         self.assertIn('cache', response)
         self.assertIn('storage', response)
         self.assertIn('permission', response)
-        self.assertEqual(response['oauth'], True)
         self.assertEqual(response['cache'], True)
         self.assertEqual(response['storage'], True)
         self.assertEqual(response['permission'], True)
```

Comment:
https://kinto.dev.mozaws.net/v1/__heartbeat__ shows a `oauth` key. Not sure if there is an easier way to compare JSON objects in Python. It may be easier to create an expected response object and just compare it to the actual response, since this test won't ever detect if new keys are added to the response.

In Node land, I'd use something like [Joi](https://github.com/hapijs/joi) to create and validate a schema, but not sure what the best way in Python-land is. (https://github.com/monsur/jsoncompare looks promising -- or maybe i'm mentally over-optimizing something that needn't be optimized.)